sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/LogicalPlan.scala

```diff
@@ -344,7 +344,8 @@ abstract class UnaryNode extends LogicalPlan {
       sizeInBytes = 1
     }
 
-    child.stats(conf).copy(sizeInBytes = sizeInBytes)
+    // Don't propagate rowCount and attributeStats, since they are not estimated here.
```

Comment:
If we remove this, estimation result of aggregate still has wrong rowCount and attributeStats.
Shall we merge this and I'll do tests for other unaryNodes and fix them if something still goes wrong.